scenes/game/local_game.gd

```diff
@@ -1466,6 +1516,7 @@ class Player:
 	func on_position_changed(old_pos, buddy_old_pos, is_self_move):
 		if is_self_move and parent.active_strike:
 			moved_self_this_strike = true
+			spaces_moved_this_strike += abs(arena_location - old_pos)
```

Comment:
Interesting thought, if you advance 1 through the opponent does that count as moving 2 spaces?
I assume it probably does, but maybe double check?
As coded it does.

Reply:
Apparently it doesn't rip